### PR TITLE
HITRAN molmass

### PR DESCRIPTION
**Pull request by ykawashima**
I have added an option to use the molecular mass provided by the HITRAN database;
https://hitran.org/docs/iso-meta/

Thank you in advance.

### CODE REVIEW

**Comment by HajimeKawahara** [comment]
Thanks! Can you add a short unit test?
for instance to check

```python
molmass('CO2',db_HIT=True)
```

**Comment by ykawashima** [comment]
I have added tests/unittests/atom/molmass_test.py. This is the first time for me to write a unit test. Is the code fine with you?

File: tests/unittests/atom/molmass_test.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"""test for molmass calculation"""
+
+import pytest
+from exojax.spec.molinfo import molmass
+
+
+def test_molmass():
+    molmass('air')
```
Comment (erwanp) [inline tests/unittests/atom/molmass_test.py:8]: (if I can suggest, you should add `assert` statements there
ex: 
```python
assert molmass('air')== 28.97
```

Comment (erwanp) [reply tests/unittests/atom/molmass_test.py:8]: They'll raise an AssertionError if it doesn't work. 

Advanced note : sometimes (not here) you want to test that an error is raised, i.e., the normal behavior is to expect an error (for instance, you're trying to catch a wrong user input, etc.) 
You can use 
```python 
with pytest.raises(ValueError):
    this_code()   #  test will pass IF this_code() raises a ValueError
```

Comment (ykawashima) [reply tests/unittests/atom/molmass_test.py:8]: Thank you very much for your suggestion and helpful tip! I will implement it.

**Review by HajimeKawahara — APPROVED** [review]
Thank you! All of the unit tests passed.

**Pull request merged**